gneiss/regression/tests/test_format.py

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import shutil
+import unittest
+import os
+from gneiss.regression._format import (RegressionFormat_g,
+                                       RegressionDirectoryFormat_g)
+from qiime2.plugin.testing import TestPluginBase
+
+
+class TestFormats(TestPluginBase):
+    package = "gneiss.regression.tests"
+
+    def test_regression_format_validate_positive(self):
+        filepath = self.get_data_path('ols.pickle')
+        format = RegressionFormat_g(filepath, mode='r')
+
+        print(filepath, str(format), format.sniff())
+        format.validate()
```

Comment:
I still new to unittests in Python. Don't you have to add some kind of self.assertxxx statements to the code?